extensions/amp-video-docking/0.1/controls.js

```diff
@@ -427,20 +415,27 @@ export class Controls {
 
   /** @private */
   showOnNextAnimationFrame_() {
```

Comment:
1. `mousemove` listener is removed when controls hide. is this a problem?
2. Verify what particularly? I'd like to display with new state each time rather than mutate. It's simpler and I don't think it's particularly more expensive.